src/ec/packages/ec-component-ecl-compliance/components/ecl-compliance-page-header-core.html.twig

```diff
@@ -0,0 +1,21 @@
+{#
+  Parameters:
+    title: (string),
+    breadcrumb: (object)
+#}
+
+{% extends "@ecl-twig/ec-component-ecl-compliance/ecl-compliance.html.twig" %}
+
+{% set message = '' %}
+{% set component = 'page-header-core' %}
+{% set no_title_case = title is empty %}
+{% set no_breadcrumb_case = breadcrumb is empty or breadcrumb.links is not defined %}
```

Comment:
Something is not quite right here

![missing_breadcrumb](https://user-images.githubusercontent.com/1923476/82782953-c60b5500-9e5d-11ea-8331-e2c32b8b68ba.png)

Reply:
Same behavior appears in https://github.com/ec-europa/ecl-twig/pull/456

Reply:
@kalinchernev , what you are experiencing is a validation performed by storybook on the input data, in this case we define an object knob, emptying the text-area fires this "error" and marks the knob as invalid, with the red border. The story doesn't get updated when you do this and indeed you still have the breadcrumb in place in the demo.

If you want to test the breadcrumb case, you have to put `{} ` which, defining an empty object, is accepted by storybook and triggers the warning by our validation, as we do in the test.